draft-ietf-ppm-dap.md

```diff
@@ -2097,11 +2066,8 @@ the resulting PlaintextInputShare `plaintext_input_share`.
 
 #### Input Share Validation {#input-share-validation}
 
-Validating an input share will either succeed or fail. In the case of failure,
-the input share is marked as invalid with a corresponding report error.
-
```

Comment:
brevity: first senctence is tautological and the rest is clear from the remainder of the section